### DISCUSSION TITLE
Displaying data from two db tables into one table builder

### DISCUSSION THREAD

**Discussion by w0r00d** [filamentphp/filament#14467] [Help] [answered]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

Hi!

I have two tables that have identical structure and data types and I want to display the data of the two tables in one table in filament.

I tried using unionAll() in the query section but it shows an error.

As follows

`  return $table
            ->query(
                PendingBeneficiary::query()->unionAll(CashBeneficiary::query())`

But I'm getting this error

![error](https://github.com/user-attachments/assets/e5711b90-d62a-43c5-9e6b-7c6fcae0e98d)

It's weird because I am not using order by statement, is there another way to accomplish this or fix this error?

**Comment by leandrocfe** [comment, marked as answer]
I think you could create a [DB View](https://stackoverflow.com/questions/26593270/how-to-access-db-views-using-laravel-models), add the union and use this DB View as a Model.

**Reply by w0r00d** [reply]
That actually worked like magic and is much more practical than the complicated queries i was attempting!

Thank you so much for reminding me of views, never occurred to me.
